### PR TITLE
AW-11 | change structure gradeComposition and gradeStructure.

### PR DESCRIPTION
**Pull request by anhkhoajjo**

### CODE REVIEW

**Comment by anhkhoajjo** [comment]
[Change gradeComposition and gradeStructure entity](https://trello.com/c/pWGoIGh3/11-change-gradecomposition-and-gradestructure-entity)

File: src/main/webapp/i18n/en/global.json [hunk 1]
```diff
@@ -25,8 +25,7 @@
         "sessions": "Sessions",
         "login": "Sign in",
         "logout": "Sign out",
-        "register": "Register",
-        "profile": "Profile"
```
Comment (huykevin2002) [inline src/main/webapp/i18n/en/global.json:29]: revert pls

File: src/main/webapp/i18n/en/global.json [hunk 2]
```diff
@@ -107,8 +106,7 @@
       "edit": "Edit",
       "open": "Open",
       "save": "Save",
-      "view": "View",
-      "show": "Show {{otherEntity}}"
```
Comment (huykevin2002) [inline src/main/webapp/i18n/en/global.json:111]: revert pls

File: src/main/webapp/i18n/en/global.json [hunk 3]
```diff
@@ -129,11 +127,6 @@
       "number": "This field should be a number.",
       "datetimelocal": "This field should be a date and time.",
       "patternLogin": "This field can only contain letters, digits and e-mail addresses."
-    },
-    "filters": {
-      "set": "Following filters are set",
-      "clear": "Clear filter",
-      "clearAll": "Clear all filters"
     }
```
Comment (huykevin2002) [inline src/main/webapp/i18n/en/global.json:130]: revert pls

File: src/main/webapp/i18n/en/gradeComposition.json [hunk 4]
```diff
@@ -19,14 +19,13 @@
       },
       "id": "ID",
       "name": "Name",
-      "minGradeScale": "Min Grade Scale",
-      "maxGradeScale": "Max Grade Scale",
-      "position": "Position",
```
Comment (huykevin2002) [inline src/main/webapp/i18n/en/gradeComposition.json:24]: revert if necessary

File: src/main/webapp/i18n/en/password.json [hunk 5]
```diff
@@ -2,7 +2,7 @@
   "password": {
     "title": "Password for [<strong>{{username}}</strong>]",
     "form": {
-      "button": "Save change"
```
Comment (huykevin2002) [inline src/main/webapp/i18n/en/password.json:5]: revert please

File: src/main/webapp/i18n/en/settings.json [hunk 6]
```diff
@@ -1,17 +1,13 @@
 {
   "settings": {
-    "title": "{{username}} Profile",
-    "description": "Manage your profile settings",
-    "general": "General",
-    "security": "Security",
```
Comment (huykevin2002) [inline src/main/webapp/i18n/en/settings.json:6]: revert please

File: src/main/webapp/i18n/en/settings.json [hunk 7]
```diff
@@ -1,17 +1,13 @@
...
+    "title": "User settings for [<strong>{{username}}</strong>]",
     "form": {
       "firstname": "First Name",
       "firstname.placeholder": "Your first name",
       "lastname": "Last Name",
       "lastname.placeholder": "Your last name",
       "language": "Language",
-      "button": "Save",
-      "saveChanges": "Save change"
```
Comment (huykevin2002) [inline src/main/webapp/i18n/en/settings.json:14]: revert please

File: src/main/webapp/i18n/vi/global.json [hunk 8]
```diff
@@ -25,8 +25,7 @@
         "sessions": "Phiên đăng nhập",
         "login": "Đăng nhập",
         "logout": "Đăng xuất",
-        "register": "Đăng ký",
-        "profile": "Hồ sơ"
```
Comment (huykevin2002) [inline src/main/webapp/i18n/vi/global.json:29]: revert please

File: src/main/webapp/i18n/vi/global.json [hunk 9]
```diff
@@ -45,8 +44,6 @@
     "form": {
       "username.label": "Tên tài khoản",
       "username.placeholder": "Nhập tên tài khoản của bạn",
-      "currentpassword.label": "Mật khẩu hiện tại",
-      "currentpassword.placeholder": "Nhập mật khẩu hiện tại",
```
Comment (huykevin2002) [inline src/main/webapp/i18n/vi/global.json:49]: revert please

File: src/main/webapp/i18n/vi/global.json [hunk 10]
```diff
@@ -107,8 +104,7 @@
       "edit": "Sửa",
       "open": "Mở",
       "save": "Lưu",
-      "view": "Xem",
-      "show": "Show {{otherEntity}}"
```
Comment (huykevin2002) [inline src/main/webapp/i18n/vi/global.json:111]: revert please

File: src/main/webapp/i18n/vi/global.json [hunk 11]
```diff
@@ -128,11 +124,6 @@
       "pattern": "Trường này phải theo định dạng {{ pattern }}.",
       "number": "Trường này phải là một số.",
       "datetimelocal": "Trường này phải là định dạng thời gian."
-    },
-    "filters": {
-      "set": "Following filters are set",
-      "clear": "Clear filter",
-      "clearAll": "Clear all filters"
```
Comment (huykevin2002) [inline src/main/webapp/i18n/vi/global.json:135]: revert please

File: src/main/webapp/i18n/vi/gradeComposition.json [hunk 12]
```diff
@@ -19,14 +19,13 @@
       },
       "id": "ID",
       "name": "Name",
-      "minGradeScale": "Min Grade Scale",
-      "maxGradeScale": "Max Grade Scale",
-      "position": "Position",
```
Comment (huykevin2002) [inline src/main/webapp/i18n/vi/gradeComposition.json:24]: revert if necessary

File: src/main/webapp/i18n/vi/login.json [hunk 13]
```diff
@@ -3,7 +3,7 @@
     "title": "Đăng nhập",
     "form": {
       "password": "Mật khẩu",
-      "password.placeholder": "Nhập mật khẩu của bạn",
```
Comment (huykevin2002) [inline src/main/webapp/i18n/vi/login.json:6]: revert pls

File: src/main/webapp/i18n/vi/password.json [hunk 14]
```diff
@@ -2,7 +2,7 @@
   "password": {
     "title": "Mật khẩu cho tài khoản [<strong>{{username}}</strong>]",
     "form": {
-      "button": "Lưu thay đổi"
```
Comment (huykevin2002) [inline src/main/webapp/i18n/vi/password.json:5]: revert pls

File: src/main/webapp/i18n/vi/settings.json [hunk 15]
```diff
@@ -1,17 +1,13 @@
 {
   "settings": {
-    "title": "Hồ sơ của {{username}} ",
-    "description": "Quản lý thông tin cá nhân",
-    "general": "Thông tin chung",
-    "security": "Bảo mật",
```
Comment (huykevin2002) [inline src/main/webapp/i18n/vi/settings.json:6]: revert pls

File: src/main/webapp/i18n/vi/settings.json [hunk 16]
```diff
@@ -1,17 +1,13 @@
...
+    "title": "Cài đặt người dùng của [<strong>{{username}}</strong>]",
     "form": {
       "firstname": "Tên",
       "firstname.placeholder": "Nhập tên của bạn",
       "lastname": "Họ",
       "lastname.placeholder": "Nhập họ của bạn",
       "language": "Ngôn ngữ",
-      "button": "Lưu",
-      "saveChanges": "Lưu thay đổi"
```
Comment (huykevin2002) [inline src/main/webapp/i18n/vi/settings.json:14]: revert pls

**Review by huykevin2002 — COMMENTED** [review]
Review

File: src/main/webapp/i18n/en/global.json (hunk 1 above)
Comment (anhkhoajjo) [reply src/main/webapp/i18n/en/global.json:29]: I reverted it

File: src/main/webapp/i18n/en/global.json (hunk 2 above)
Comment (anhkhoajjo) [reply src/main/webapp/i18n/en/global.json:111]: I reverted it

File: src/main/webapp/i18n/en/global.json (hunk 3 above)
Comment (anhkhoajjo) [reply src/main/webapp/i18n/en/global.json:130]: I reverted it

File: src/main/webapp/i18n/en/gradeComposition.json (hunk 4 above)
Comment (anhkhoajjo) [reply src/main/webapp/i18n/en/gradeComposition.json:24]: this is necessary. so I will not revert it

File: src/main/webapp/i18n/en/password.json (hunk 5 above)
Comment (anhkhoajjo) [reply src/main/webapp/i18n/en/password.json:5]: I reverted it

File: src/main/webapp/i18n/en/settings.json (hunk 6 above)
Comment (anhkhoajjo) [reply src/main/webapp/i18n/en/settings.json:6]: I reverted it

File: src/main/webapp/i18n/en/settings.json (hunk 7 above)
Comment (anhkhoajjo) [reply src/main/webapp/i18n/en/settings.json:14]: I reverted it

File: src/main/webapp/i18n/vi/global.json (hunk 8 above)
Comment (anhkhoajjo) [reply src/main/webapp/i18n/vi/global.json:29]: I reverted it

File: src/main/webapp/i18n/vi/global.json (hunk 9 above)
Comment (anhkhoajjo) [reply src/main/webapp/i18n/vi/global.json:49]: I reverted it

File: src/main/webapp/i18n/vi/global.json (hunk 10 above)
Comment (anhkhoajjo) [reply src/main/webapp/i18n/vi/global.json:111]: I reverted it

File: src/main/webapp/i18n/vi/global.json (hunk 11 above)
Comment (anhkhoajjo) [reply src/main/webapp/i18n/vi/global.json:135]: I reverted it

File: src/main/webapp/i18n/vi/gradeComposition.json (hunk 12 above)
Comment (anhkhoajjo) [reply src/main/webapp/i18n/vi/gradeComposition.json:24]: No, this is necessary

File: src/main/webapp/i18n/vi/login.json (hunk 13 above)
Comment (anhkhoajjo) [reply src/main/webapp/i18n/vi/login.json:6]: I reverted it

File: src/main/webapp/i18n/vi/password.json (hunk 14 above)
Comment (anhkhoajjo) [reply src/main/webapp/i18n/vi/password.json:5]: I reverted it

File: src/main/webapp/i18n/vi/settings.json (hunk 15 above)
Comment (anhkhoajjo) [reply src/main/webapp/i18n/vi/settings.json:6]: I reverted it

File: src/main/webapp/i18n/vi/settings.json (hunk 16 above)
Comment (anhkhoajjo) [reply src/main/webapp/i18n/vi/settings.json:14]: I reverted it

**Review by huykevin2002 — APPROVED** [review]

**Pull request merged**